tools/cldr-unittest/src/org/unicode/cldr/unittest/TestCheckCLDR.java

```diff
@@ -457,6 +460,52 @@ public void TestCheckNew() {
         errln("No failure message.");
     }
 
+    public void TestCheckNewRootFailure() {
```

Comment:
This is the only new part. We check to make sure we return errors on both E10-386 (root emoji codes) and ^^^